crates/pipeline_manager/src/db/mod.rs

```diff
@@ -246,87 +241,42 @@ pub(crate) struct ProjectDescr {
     pub schema: Option<String>,
 }
 
-/// Project configuration descriptor.
+/// PipelineDescr descriptor.
 #[derive(Serialize, ToSchema, Eq, PartialEq, Debug, Clone)]
-pub(crate) struct ConfigDescr {
-    pub config_id: ConfigId,
-    pub project_id: Option<ProjectId>,
-    pub pipeline: Option<PipelineDescr>,
+pub(crate) struct PipelineDescr {
+    pub pipeline_id: PipelineId,
+    pub program_id: Option<ProgramId>,
     pub version: Version,
     pub name: String,
     pub description: String,
     pub config: String,
```

Comment:
Now that config is being saved in the DB, should we turn it into a json object all the way through including here?

Reply:
I think it's toml but it was always in the DB. I will try to make it a toml object in next PR.